aten/src/ATen/native/native_functions.yaml

```diff
@@ -6343,19 +6343,38 @@
     CPU: sort_out_cpu
     CUDA: legacy::cuda::_th_sort_out
 
+- func: sort.values_stable(Tensor self, *, bool? stable, int dim=-1, bool descending=False, Tensor(a!) values, Tensor(b!) indices) -> (Tensor(a!) values, Tensor(b!) indices)
+  use_c10_dispatcher: hacky_wrapper_for_legacy_signatures
+  dispatch:
+    CPU: sort_out_cpu_stable
+    CUDA: legacy::cuda::_th_sort_out_stable
```

Comment:
@mruberry , @glaringlee , are you fine with these schemas? I had to make `bool?`  instead of `bool` because some methods in ATen call to `sort(int)`, and this confuses compiler with the overloaded function selection.

Reply:
@mruberry 
This seems fine to me, any advise?